NPE related with Wirtschaftszweig Combobox

- Import address data with three different "Wirtschaftszweig" entries respectively without "Wirtschaftszweig" using the following two additional lines between the existing ones in data/db/import.csv:
```csv
Beispielklassifizerung,XXX,Muster AG,Mustermann,Max,max.mustermann@example.test,01234-123123,01234-321321,33602,Bielefeld,August-Bebel-Straße,75,,Bemerkung1
Beispielklassifizerung,,Muster KG,Mustermann,Max,max.mustermann@example.test,01234-123123,01234-321321,,,,,,Bemerkung1
```
- Start application with `mvn clean compile exec:java`
- Click "Betreiber/Inhaber suchen"
- Click "Alle Adressen"
- Double-click entry for "Muster KG"
=>
```
java.lang.RuntimeException: SwingWorker error in doUIUpdateLogic
    at de.bielefeld.umweltamt.aui.utils.SwingWorkerVariant.finished (SwingWorkerVariant.java:181)
    at de.bielefeld.umweltamt.aui.utils.SwingWorkerVariant$1.run (SwingWorkerVariant.java:85)
    at java.awt.event.InvocationEvent.dispatch (InvocationEvent.java:313)
[...]
Caused by: java.lang.NullPointerException
    at de.bielefeld.umweltamt.aui.module.common.editors.BetreiberEditor$BetreiberListener.actionPerformed (BetreiberEditor.java:1082)
    at javax.swing.JComboBox.fireActionEvent (JComboBox.java:1264)
[...]
```

It seems also that sometimes, if the application is first started and the import run afterwards, additional imported "Wirtschaftszweig" entries do not appear in the combobox.

--- a/README.md
+++ b/README.md
@@ -103,3 +103,7 @@ psql -d fisumwelt -f import_csv.sql <import.csv
 ```
 
 `fisumwelt` ist hier der Datenbankname, je nach Datenbank-Setup müssen ggf. weitere Parameter ergänzt werden.
+
+Manche der importierten Daten werden nur beim Start der Anwendung geladen,
+so dass ein Neustart der Anwendung nach dem Import erforderlich ist,
+damit alle Daten korrekt dargestellt werden.

